build/secondary/testing/gtest/BUILD.gn

```diff
@@ -18,7 +18,10 @@ config("gtest_config") {
   ]
 
   # Gtest headers need to be able to find themselves.
-  include_dirs = [ "include" ]
+  include_dirs = [
```

Comment:
That would mean looking at "//build/secondary/testing/gtest/include". There is no such path. The gtest checked out by deps is in //testing/gtest